FireRender.Maya.Src/scripts/registerFireRender.mel

```diff
@@ -10,7 +10,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-source "createMayaSoftwareCommonGlobalsTab.mel";
+//source "createMayaSoftwareCommonGlobalsTab.mel";
```

Comment:
Please remove comment or uncomment if this is necessary

Reply:
Done